docs/using/recipes.md

```diff
@@ -516,3 +516,24 @@ RUN pip install jupyter_contrib_nbextensions && \
 
 Ref:
 [https://github.com/jupyter/docker-stacks/issues/675](https://github.com/jupyter/docker-stacks/issues/675)
+
+## Enable auto-sklearn notebooks
+
+Using auto-sklearn requires swig, which the other notebook images lack, so it cant be experimented with. Also, there is no Conda package for auto-sklearn.
+
+```
+ARG BASE_CONTAINER=jupyter/scipy-notebook
+FROM jupyter/scipy-notebook:latest
+
+USER root
+
+# autosklearn requires swig, which no other image has
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends swig && \
+    rm -rf /var/lib/apt/lists/*
+
+USER $NB_UID
+ARG py_ver=3.6
+
+RUN pip install auto-sklearn
```

Comment:
`pip install --quiet`

Reply:
Maybe the install could break permissions. Could you check it and if needed add

```dockerfile
 fix-permissions $CONDA_DIR && \
 fix-permissions /home/$NB_USER
```

Reply:
I tested this again, the install did not seem to break perms, I added the quiet as well. Thank you for the input.